holidata/holidays/hr-HR.py

```diff
@@ -7,18 +7,19 @@
 class hr_HR(Locale):
     """
     01-01: [NF] Nova Godina
-    01-06: [NRF] Bogojavljanje / Sveta tri kralja
+    01-06: [NRF] Bogojavljanje ili Sveta tri kralja
     05-01: [NF] Praznik rada
     06-22: [NF] Dan antifašističke borbe
     06-25: [NF] Dan državnosti
-    08-05: [NF] Dan domovinske zahvalnosti
+    08-05: [NF] Dan pobjede i domovinske zahvalnosti i Dan hrvatskih branitelja
     08-15: [NRF] Velika Gospa
     10-08: [NF] Dan neovisnosti
-    11-01: [NRF] Dan svih svetih
+    11-01: [NRF] Svi sveti
```

Comment:
OK: Use official name for "Svi sveti" (All Saint's Day) as mentioned in the respective law

Reply:
> OK: Use official name for "Svi sveti" (All Saint's Day) as mentioned in the respective law

Yes.